cypress/support/dnd.ts

```diff
@@ -199,14 +199,14 @@ Cypress.Commands.add(
         });
 
       if (position === 'inside') {
-        cy.get('@target').trigger('dragenter', {
+        cy.get('@target').trigger('dragover', {
```

Comment:
Since this is going to fire on every tick of a drag event, is there any measurable way to check how this might affect the overall dnd performance?

Reply:
There's some optimisations in the `Positioner` class to prevent the indicator being updated on every tick, so that helps quite a bit. 

Not familiar with an automated way test this  - it's difficult to test dnd with Cypress because the best we can do there is emulate dnd events. We could just check the FPS count manually, and this new implementation is more or less the same as the previous one, so that should be good enough.